.github/workflows/build.yml

```
@@ -0,0 +1,37 @@
+name: "Build test build on PRs"
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+          cache: "pip"
+      - name: Install dependencies
+        run: pip install -r requirements.txt
+      - name: Install colloid specific dependencies
+        run: sudo apt update && sudo apt install -y sassc inkscape optipng
+      - name: Generate themes
+        run: |
+          python ./build.py mocha --all-accents --zip -d $PWD/releases &&
+          python ./build.py macchiato --all-accents --zip -d $PWD/releases &&
+          python ./build.py frappe --all-accents --zip -d $PWD/releases &&
+          python ./build.py latte --all-accents --zip -d $PWD/releases
```

If there's a failure with only one themes (unlikely) then its worth not using the `&&` syntax
```suggestion
          python ./build.py mocha --all-accents --zip -d $PWD/releases
          python ./build.py macchiato --all-accents --zip -d $PWD/releases
          python ./build.py frappe --all-accents --zip -d $PWD/releases;
          python ./build.py latte --all-accents --zip -d $PWD/releases
```